src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -61,6 +62,7 @@ const PreviewCard = ({
             </small>
           </p>
           <div className="pop-up-description rounded">{display_name}</div>
```

Comment:
Synonym or Synonyms?

Reply:
@dkayiwa I now get it. I thought I only had to display one synonym. Let me display all the synonyms of the concept.